cmd/crowdsec/serve.go

```diff
@@ -227,6 +229,24 @@ func Serve(cConfig *csconfig.Config) error {
 	apiTomb = tomb.Tomb{}
 	crowdsecTomb = tomb.Tomb{}
 	pluginTomb = tomb.Tomb{}
+
+	if cConfig.API.Server != nil && cConfig.API.Server.DbConfig != nil {
+		dbClient, err := database.NewClient(cConfig.API.Server.DbConfig)
+		if err != nil {
+			return errors.Wrap(err, "Failed to get database client")
```

Comment:
Sorry to be annoying, but we shouldn't capitalize error messages : https://github.com/golang/go/wiki/CodeReviewComments#error-strings (applies to bellow error messages as well)